crates/fluvio-protocol/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "fluvio-protocol"
 edition = "2021"
-version = "0.9.0"
+version = "0.10.0"
```

Comment:
Don't think this need major bump since we are only adding new API?

Reply:
My understanding is that what I did here is a minor bump.

I think you are right though. The SemVer docs say once the major version is >0, API additions require a minor version bump.

But since the major version is still 0, I think it is still subjective. I can change these all to increment the patch value instead, if you like.

Reply:
Yes, let's change to only patch version which should  minimize impact to downstream crates